M2/Macaulay2/m2/set.m2

```diff
@@ -81,9 +81,16 @@ Set.synonym = "set"
 
 -- constructor
 new Set from List := Set => (X,x) -> set x -- compiled function
-elements Set := List => x -> keys x
+
+-- compiled functions which will be documented
+-- TODO: why doesn't this work?
+-- # Set        := Boolean => lookup(symbol #,  HashTable)
+-- Set #? Thing := Boolean => lookup(symbol #?, HashTable, Thing)
+installMethod(symbol #,  Set,        Boolean => x -> x)
+installMethod(symbol #?, Set, Thing, Boolean => x -> x)
```

Comment:
Should these installed methods raise an error like the `dummy` ones in `typicalvalues.m2`?

Reply:
I don't follow. Raise an error for what?

Reply:
I was just suggesting that maybe for consistency, the installed method should be `x -> (dummy x;)`, where `dummy` is defined as `x -> error("dummy method function called")` like the other ones in typicalvalues.m2.

Why add these here rather than a comment in the d directory?  And do we even need to bother since `Set` inherits from `HashTable`, and it's already registered using a comment in actors.d?

Also, wouldn't the typical value for `# Set` be `ZZ` instead of `Boolean`?